samza-core/src/test/java/org/apache/samza/clustermanager/TestContainerProcessManager.java

```diff
@@ -226,6 +227,56 @@ public void run() {
     cpm.stop();
   }
 
+  @Test
+  public void testOnInitAMHighAvailability() throws Exception {
+    configVals.put(JobConfig.YARN_AM_HIGH_AVAILABILITY_ENABLED, "true");
```

Comment:
Seems like you are modifying a static map here and the remnants of the test is going to impact other tests in the class.

Can we make a copy instead and add AM-HA configs?